### PR TITLE
feat: new template for batch execution

### PR DESCRIPTION
**Pull request by TaoSunVoyage**
Add template for batch execution #118

Main difference between python-minimal template:
- no `environment.yml`, python dependencies have to be installed by pip
- `Dockerfile.batch` for batch version image, and a new job in `.gitlab-ci.yml`
- `sbatch-script-template.sh` template for sbatch script

### CODE REVIEW

**Comment by TaoSunVoyage** [comment]
The content and location of `sbatch-script-template.sh` is still open for discussion.

**Review by rokroskar — COMMENTED** [review]
Nice, thanks for putting this together! a few minor comments

File: batch-minimal/Dockerfile.batch [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# For finding latest versions of the base image see
+# https://github.com/SwissDataScienceCenter/renkulab-docker
+ARG RENKU_BASE_IMAGE=renku/renkulab-batch
```
[the remark this answers is not in the document]
Comment (TaoSunVoyage) [reply batch-minimal/Dockerfile.batch:3]: yes. probably 0.8.x in the next releases of renku-docker

File: batch-minimal/sbatch_script_template.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -l
```
[the remark this answers is not in the document]
Comment (TaoSunVoyage) [reply batch-minimal/sbatch_script_template.sh:1]: well, yes. but i am still not sure what would be the best for this script-template. content, location, etc. some decisions to be made.

File: batch-minimal/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# {{ name }}
```
[the remark this answers is not in the document]
Comment (TaoSunVoyage) [reply batch-minimal/README.md:1]: yes. absolutely.

**Comment by ableuler** [comment]
@TaoSunVoyage is this closed to being merged? I think we're going to tag a new release here soon.

**Comment by TaoSunVoyage** [comment]
@ableuler not yet. We are waiting for the new workflow executor `toil`.

**Comment by rokroskar** [comment]
I'm not sure we need a separate template for batch execution. I rather envision a situation where a "batch" dockerfile (and associated configuration) is added to a normal project. Closing this for now, we may revisit at some later point.

**Pull request closed without merging**